docs/ARCH-PS-Data-Layer.md

```diff
@@ -11,39 +11,39 @@ Profile server uses Entity Framework Core ORM on SQLite database with code-first
 
 ### Database Models and Repositories
 
-Each database table is represented by its model class and is accessed through its repository. The following database tables are used:
+Each database table is represented by its model class and it's accessed through its repository. The following database tables are used:
 
  * Settings - Stores part of the configuration of the profile server including the profile server cryptographic identity.
  * Identities - Stores all identities that are hosted on this profile server. All identity profile data are stored in this table 
 except for the profile images that are stored outside the database.
  * NeighborIdentities - Stores all identities that are hosted on neighbor profile servers. All identity profile data are stored in this table 
 except for the thumbnail image that is stored outside the database.
- * RelatedIdentities - Stores information about relationships between profiles. These relationships are announced by the identities themselves, but they has to be cryptographically proved to be accepted.
+ * RelatedIdentities - Stores information about relationships between profiles. These relationships are announced by the identities themselves, but they have to be cryptographically proved to be accepted.
  * NeighborhoodActions - Lists of pending actions related to the management of the profile server's neighborhood are stored in this table. When a profile server is informed about a change 
-in its neighborhood, such as a new server joined the neighborhood, or a server left the neighborhood, a new action is added. Similarly, in case of a change in the profile server's hosted 
+in its neighborhood, such as if a new server joined the neighborhood, or a server left the neighborhood, a new action is added. Similarly, in case of a change in the profile server's hosted 
 profiles database, the change has to be propagated to the followers, which is done through actions in this table.
  * Neighbors - Stores a list of profile servers that the profile server considers to be its neighbors, i.e. updates of their hosted profiles can come from them.
  * Followers - Stores a list of profile servers that are following the profile server, i.e. changes in hosted profiles has to be propagated to them.
 
 
 ### Database Component
 
-Database component cares about maintaining the database in a good shape, which means that it performs a number of cleanup tasks 
-both during the startup as well as periodically during the normal life of profile server. These tasks remove unused records from 
+The Database component cares about maintaining the database in a good shape, which means that it performs a number of cleanup tasks 
```

Comment:
yes, i probably got mixed up because i thought it was starting a sentence.. so yes it should be lower case 'd' also the 't'